### PR TITLE
New LL Card - Square Card

### PR DESCRIPTION
**Pull request by iantrich**
## Proposed change
Started working on a design inspired by Apple's HomeKit here: https://www.figma.com/file/cfEcM4wLZ8N4kBaAoalDzN/HomeKit-Style-Button?node-id=0%3A1
This is my attempt at putting that into code
Preview
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -421,6 +421,13 @@
       "relative_time": {
         "never": "Never",
         "just_now": "Just now",
+        "abbreviated_duration": {
+          "second": "{count}s",
+          "minute": "{count}m",
+          "hour": "{count}h",
+          "day": "{count}d",
+          "week": "{count}w"
+        },
```
Comment (spacegaier) [inline src/translations/en.json:430]: I think there should a be space between value and unit to be consistent with the rest of Lovelace.

**Comment by iantrich** [comment]
Also, let's talk design first before we even look at my code ;) I threw this together and know it is not pretty code-wise

File: src/translations/en.json (hunk 1 above)
Comment (iantrich) [reply src/translations/en.json:430]: Personally, I think that would look off and don't believe this should necessarily conform to the rest of Lovelace.

**Comment by bramkragten** [comment]
Should we do a meeting to brainstorm about this?

**Comment by iantrich** [comment]
Yes

**Comment by zsarnett** [comment]
Looks really good. Was there ever a decision/discussion on this?

**Comment by iantrich** [comment]
> Looks really good. Was there ever a decision/discussion on this?

Not yet. Summertime always kills my drive to do dev work 😜

**Comment by matthiasdebaat** [comment]
@iantrich This is just freaking awesome! I'm going to schedule that design appointment. Who wants to join?

**Comment by matthiasdebaat** [comment]
Some meeting notes:
- Keep the font size icon the same if the card is larger than designed.
- What color should it be when the light is off or unavailable? Off is blue and gray not available?
- Top right info is not in the MVP. In a next release it can be empty by default. The user can place extra info there, such as an attribute.
- Top right info for light and thermostat set? Or can that be done differently?
- We need to make an example for RTL.
- What should we do with the media player?

**Comment by matthiasdebaat** [comment]
Just made another iteration on the design: https://www.figma.com/file/nZjYhfqXnw9buKkIJ628D1/Square-Card?node-id=401%3A294

If we go to scale the card, like we do with the other Lovelace cards, it doesn't look very good. In the design I made two examples. One is too big and the other too small. Is there any way to make this better?

**Comment by bramkragten** [comment]
We can make the size of the icon a little bit dynamic maybe, so it would be smaller on a small card, and would fit better?

**Pull request closed without merging**